src/routes/chat/+page.svelte

```diff
@@ -220,6 +221,10 @@
         await RaygunStoreInstance.downloadAttachment($conversation!.id, message, attachment.name, attachment.size)
     }
 
+    async function typing() {
+        await RaygunStoreInstance.sendEvent($activeChat.id, MessageEvent.Typing)
+    }
```

Comment:
I wonder if we should put this into a try/catch in case any uncaught exceptions are ignored.